packages/appstore/src/components/appstore-banner/svg-migration-banner.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Button, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
+import AppstoreBannerImage from './appstore-banner-image';
+
+const SVGMigrationBanner = observer(() => {
+    const {
+        ui,
+        modules: { cfd },
+    } = useStore();
+    const {
+        eligible_svg_to_bvi_derived_accounts,
+        eligible_svg_to_bvi_financial_accounts,
+        eligible_svg_to_vanuatu_derived_accounts,
+        eligible_svg_to_vanuatu_financial_accounts,
```

Comment:
```suggestion
        eligible_svg_to_bvi_derived_accounts,
        eligible_svg_to_bvi_financial_accounts,
        eligible_svg_to_vanuatu_derived_accounts,
        eligible_svg_to_vanuatu_financial_accounts,
```

You can encapsulate this within `useMT5SVGEligibleToMigrate()` instead of calculating the conditions as per below